### PR TITLE
upgrade to SBT version 1.1.0

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

**Comment by sullis** [comment]
please review @dwijnand

File: build.sbt [hunk 1]
```diff
@@ -11,9 +11,9 @@ organization := "com.dwijnand"
 
        sbtPlugin           := true
       sbtVersion in Global := "0.13.16" // must be Global, otherwise ^^ won't change anything
-crossSbtVersions           := List("0.13.16", "1.0.0-RC3")
+crossSbtVersions           := List("0.13.16", "1.1.0")
```
Comment (dwijnand) [inline build.sbt]: The rest looks good, except for this change. I'd like to test that this plugin works against versions of sbt prior to 1.1.

Now that doesn't have to be 1.0.0-RC3, so if you want you can update it to 1.0.0.

**Comment by sullis** [comment]
ready for review.

**Comment by sullis** [comment]
please merge :)

**Comment by dwijnand** [comment]
Thanks for the contribution!

**Pull request merged**